ansible_collections/arista/avd/plugins/filter/markdown_rendering.py

```diff
@@ -5,6 +5,21 @@
 
 __metaclass__ = type
 
+from ansible.errors import AnsibleFilterError
+
+from ansible_collections.arista.avd.plugins.plugin_utils.pyavd_wrappers import RaiseOnUse, wrap_filter
+
+PLUGIN_NAME = "arista.avd.status_render"
+
+try:
+    from pyavd.j2filters.markdown_rendering import status_render
```

Comment:
Let's fix the names to be consistent `markdown_rendering`

Reply:
ah my bad. The filter name is actually `status_render` so we should use that everywhere - also for filenames.

Reply:
`status_render` to `markdown_rendering` ?

Reply:
Okay, lemme change the file name

Reply:
done